detox/src/Detox.js

```diff
@@ -23,6 +24,9 @@ const DEVICE_CLASSES = {
   'android.attached': AttachedAndroidDriver,
 };
 
+const _initHandle = Symbol('_initHandle');
+const _assertNoPendingInit = Symbol('_assertNoPendingInit');
```

Comment:
What's the benefit of using a symbol here? 🤔

Reply:
Actually I want to hide these and not only these properties from the users of Detox, so there is zero chance they use unless they explicitly ask us in Issues and we consider that.

Since user can get a Detox instance through `const detoxInstance = await detox.init();`, I think this is quite important to not expose more than we really want to.

Reply:
E.g. I would hide also `_client`, `_server`, `_artifactsManager` from the external access.